tests/e2e/test_e2e.py

```diff
@@ -240,19 +239,44 @@ def test_prompt_manager_no_model_params():
         )
 
 
-def test_prompt_manager_no_model_params_undeployed():
+def test_prompt_manager_undeployed_latest_revision():
     mgr = UsedByCiDontDeleteNoParamsUndeployedPromptManager(
-        UsedByCiDontDeleteNoParamsUndeployedMinorVersion.DANGEROUSLY_USE_UNDEPLOYED,
+        # Need to use a user-scoped API key to access undeployed prompts
+        api_key=os.environ["AUTOBLOCKS_API_KEY_USER"],
+        # Request the latest revision
+        minor_version="latest",
```

Comment:
the minor_version can actually be any string and we'll always grab the latest undeployed revision, but by setting it to "latest" it'll auto-update on an interval which might be nice for local dev if you're starting a flask app or something long-running